src/main/java/com/rcv/GuiMainController.java

```diff
@@ -41,15 +41,15 @@
 
   // text area which communicates the status of the tabulator's operations
   @FXML
-  private TextArea textStatus;
+  private TextArea textAreaStatus;
 
```

Comment:
Is the textAreaStatus scrollable (or otherwise expandable) or more of a single-line status?

Reply:
Yep, TextAreas have that functionality out of the box with JavaFX. It's great!